### ISSUE TITLE
Container element is empty - Siteimprove - ckeditor live regions

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9502]
From https://github.com/uiowa/uiowa/issues/9462#issuecomment-3702748539

# Problem

CKEditor aria live wrappers report container element is empty.

Container element is empty - 1.3.1 Info and Relationships WCAG 2.1 A

ck-body-wrapper

**Comment by bspeare** [comment]
# Grooming Conversation

**Quadrant:** 
- Complex - based on unknown solution at this point. 

**Discussion:**

- Investigate Gin to see if these errors are fixed (for fun)?
- Investigate upstream issue to see if any of the patches fix this error. 

**Proposed solution or definition of done:**

- A solution in place that doesn't trigger Siteimprove errors.

**Comment by joewhitsitt** [comment]
In Drupal Slack, James Wilson mentioned:

>we’re ignoring this as a false positive in Siteimprove under the following reasoning: inserting a dummy `<li>` to satisfy structural rules would break the expected dynamic behavior of the live region and introduce unnecessary noise for assistive technologies.

**Comment by joewhitsitt** [comment]
to fork an admin theme template like claro, i guess you need to sub theme it which is why attempts to register an override template in uiowa_core were not successful.

**Comment by joewhitsitt** [comment]
Changed size since issue was split in half. https://github.com/uiowa/uiowa/issues/9563
